internal/appmain/appmain.go

```diff
@@ -89,6 +92,18 @@ func (b *Bindings) AddHealthCheckFunc(f func(context.Context) error) {
 	b.sp.AddHealthCheckFunc(f)
 }
 
+// RegisterViews begins collecting data for the given views.
+func (b *Bindings) RegisterViews(v ...*view.View) {
+	if err := view.Register(v...); err != nil && b.firstErr == nil {
```

Comment:
This will add an invalid unregister even if there's an error if firstErr has been set.

Reply:
🤦